client/app/bundles/HelloWorld/containers/Scorebook.jsx

```diff
@@ -86,14 +86,26 @@ export default React.createClass({
   },
 
   setStateFromLocalStorage(callback) {
+    let state = {}
+    const storedSelectedClassroomId = window.localStorage.getItem('scorebookSelectedClassroomId')
```

Comment:
I know this is kind of unlikely, but what happens in the case where the classroom representing the stored classroom id has been archived and is no longer available to be displayed? I guess the same could be true if two teachers are sharing a computer. We should handle the id not being in the list of classrooms.

I suppose the find call should just just fail to find a match, and therefore no selected classroom is set.

Reply:
@ddmck yes, that's the logic - nothing happens if the classroom id in local storage doesn't represent one of the classroom options. would you like me to add a comment to that effect?